notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    mcmc.run(RNG, num_observations, true_x)
</pre><p>We got a lot of divergences because the data can be out of the support. I think to perform inference, you need to set a prior for <code>high</code>  such that its minimum value should be greater than all observations. E.g.</p><pre class="ql-syntax" spellcheck="false">high = numpyro.sample("high", dist.HalfNormal()) + (x.max() if x is not None else 0)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>